OsmAnd/src/net/osmand/plus/utils/OsmAndFormatter.java

```diff
@@ -438,6 +438,38 @@ public static FormattedValue getFormattedSpeedValue(float metersPerSeconds, @Non
 		return getFormattedSpeedValue(metersPerSeconds, app, mode.hasFastSpeed(), app.getSettings().SPEED_SYSTEM.getModeValue(mode));
 	}
 
+	public static float getMpSFromFormattedValue(OsmandApplication app, float value, SpeedConstants speedFormat) {
```

Comment:
These new methods looks redundant. Probably there should be better approach.

Reply:
Duplicate of ```getSpeedInMetersPerSecond```